### PR TITLE
add NIX_PATH variable to google_image_nixos_custom

### PR DESCRIPTION
**Pull request by adrian-gierakowski**
also:
update terraform-docs to latest version and regenerate docs

### CODE REVIEW

**Comment by adrian-gierakowski** [comment]
closes #12

File: google_image_nixos_custom/main.tf [hunk 1]
```diff
@@ -25,7 +31,10 @@ variable "licenses" {
 # ----------------------------------------------------
 
 data "external" "nix_build" {
-  program = ["${path.module}/nixos-build.sh", var.nixos_config]
+  program = concat(
+    ["${path.module}/nixos-build.sh", var.nixos_config],
+    var.NIX_PATH == "" || var.NIX_PATH == "-" ? [] : ["-I", var.NIX_PATH]
```
Comment (zimbatm) [inline google_image_nixos_custom/main.tf]: It needs to be set as an environment variable. `-I` only applies a single key=value pair.

Comment (adrian-gierakowski) [reply google_image_nixos_custom/main.tf]: done in 79a8ad70f8fe472607d70ddc2003172261e86e9e

**Comment by zimbatm** [comment]
awesome. thanks!

**Pull request merged**